import_export_batches/models.py

```diff
@@ -2895,60 +2944,32 @@ def import_offices_from_endorsement_json(self, batch_uri='', batch_set_id='', or
-            office_ocd_division_id = one_entry['office_ocd_division_id']
-            party = one_entry['party']
+            candidate_name = one_entry['name']
             election_day = one_entry['election_day']
             google_civic_election_id = one_entry['google_civic_election_id']
-            candidate_id = one_entry['id']
-
-            # election lookup using state & election day, and fetch google_civic_election_id
-            if not positive_value_exists(google_civic_election_id):
-                election_manager = ElectionManager()
-                election_results = election_manager.retrieve_elections_by_election_date(election_day)
-                if election_results['success']:
-                    election_list = election_results['election_list']
-                    if len(election_list) == 1:
-                        [election] = election_list
-                        election_name = election.election_name
-                        google_civic_election_id = election.google_civic_election_id
-                    else:
-                        # If multiple entries found for a given election, look up using state code and election day
-                        # if this returns multiple election entries, do not set google_civic_election_id
-                        election_results = election_manager.retrieve_elections_by_state_and_election_date(state_code,
-                                                                                                          election_day)
-                        if election_results['success']:
-                            election_list = election_results['election_list']
-                            if len(election_list) == 1:
-                                [election] = election_list
-                                election_name = election.election_name
-                                google_civic_election_id = election.google_civic_election_id
 
             if first_line:
                 # create batch_header and batch_header_map for candidate_positions
                 first_line = False
                 try:
                     batch_header = BatchHeader.objects.create(
-                        batch_header_column_000='id',
-                        batch_header_column_001='office_name',
-                        batch_header_column_002='state_code',
-                        batch_header_column_003='party',
-                        batch_header_column_004='office_ocd_division_id',
-                        batch_header_column_005='election_day',
-                        batch_header_column_006='google_civic_election_id',
+                        batch_header_column_000='office_name',
+                        batch_header_column_001='state_code',
+                        batch_header_column_002='candidate_name',
+                        batch_header_column_003='election_day',
+                        batch_header_column_004='google_civic_election_id',
```

Comment:
Hi @mnneelam I think we still can use `party` and `office_ocd_division_id` for offices even though they aren't in our current data from Democracy for America.